### PR TITLE
Crazy Cataclysm - mi-go went to the movies

### PR DESCRIPTION
**Pull request by nexusmrsep**
Apparently Mi-Go can be a great speaker of some popular culture's texts, and in its mouth those quotes become ominous and funny - perfect for the Crazy Cataclysm Mod.

Just consider where it picked up those lines, and how the context changes in it's mouth. I picked ones that would fit well with the lore, and according to how well would they sound parroted back by a mi-go.

### CODE REVIEW

**Comment by Xhuis** [comment]
Blessed PR.

File: data/mods/CrazyCataclysm/speech.json [hunk 1]
```diff
@@ -250,5 +250,253 @@
...
+  {
+    "type" : "speech",
+    "speaker" : "mon_mi_go",
+    "sound" : "\"If you don't, I will look for you, I will find you, and I will kill you.\"",
+    "volume" : 20
+  },
+  {
+    "type" : "speech",
+    "speaker" : "mon_mi_go",
+    "sound" : "\"Roads? Where we're going we don't need roads.\"",
```
Comment (DracoGriffin) [inline data/mods/CrazyCataclysm/speech.json]: `we're going we don't` -> `we're going, we don't`
Missing comma

File: data/mods/CrazyCataclysm/speech.json [hunk 2]
```diff
@@ -250,5 +250,253 @@
...
+    "volume" : 20
+  },
+  {
+    "type" : "speech",
+    "speaker" : "mon_mi_go",
+    "sound" : "\"Fasten your seatbelts. It's going to be a bumpy night.\"",
+    "volume" : 20
+  },
+  {  },
```
Comment (DracoGriffin) [inline data/mods/CrazyCataclysm/speech.json]: Extra brackets

File: data/mods/CrazyCataclysm/speech.json [hunk 3]
```diff
@@ -250,5 +250,253 @@
...
+  {
+    "type" : "speech",
+    "speaker" : "mon_mi_go",
+    "sound" : "\"You've got to ask yourself one question:  'Do I feel lucky?'  Well, do ya punk?\"",
+    "volume" : 20
+  },
+  {  },
```
Comment (DracoGriffin) [inline data/mods/CrazyCataclysm/speech.json]: Extra brackets

File: data/mods/CrazyCataclysm/speech.json [hunk 4]
```diff
@@ -250,5 +250,253 @@
...
+  {
+    "type" : "speech",
+    "speaker" : "mon_mi_go",
+    "sound" : "\"You had me at hello.\"",
+    "volume" : 20
+  },
+  {  },
```
Comment (DracoGriffin) [inline data/mods/CrazyCataclysm/speech.json]: Extra brackets

File: data/mods/CrazyCataclysm/speech.json [hunk 5]
```diff
@@ -250,5 +250,253 @@
...
+    "speaker" : "mon_mi_go",
+    "sound" : "\"There's no place like home.\"",
+    "volume" : 20
+  },
+  {
+    "type" : "speech",
+    "speaker" : "mon_mi_go",
+    "sound" : "\"It's only a flesh wound!\"",
+    "volume" : 20
+  },   {
```
Comment (DracoGriffin) [inline data/mods/CrazyCataclysm/speech.json]: New line for bracket

File: data/mods/CrazyCataclysm/speech.json (hunk 2 above)
Comment (nexusmrsep) [reply data/mods/CrazyCataclysm/speech.json]: Oh, so that's what why Jenkins was so annoyed, I thought I got them all. Thanks

**Comment by DracoGriffin** [comment]
Looks like speech.json has been linted, so you'll need to [lint it here](http://dev.narc.ro/cataclysm/format.html) in order to pass failed checks.

Basically, it's just going to remove all the spaces like so:
`    "type" : "speech",`
to
`    "type": "speech",`

**Comment by Break-Off** [comment]
Some suggestions:
"Forget it, Jake. It's Chinatown."
"Frankly, my dear, I don't give a damn."
"All work and no play makes Jack a dull boy."
"You're gonna need a bigger boat."
"Remember: all I'm offering is the truth. Nothing more."
"Don't call me Shirley."

**Comment by nexusmrsep** [comment]
> "Forget it, Jake. It's Chinatown."
> "You're gonna need a bigger boat."

I cannot think of a scenario even remotely true to the lore in which mi-go could hear those lines. Chinatown in a lab? Bigger boat in a lab? This is why I ommited them in the first place. Correct me if I'm wrong. Rest seems fine and I'll add them when I can.

**Comment by Epictyphlosion** [comment]
Some more suggestions:
"GOLDSTIEN! GOLDSTIEN!"
"Yippie-kai-yay, motherfucker"
"I have had it with these motherfucking snakes!"
"Back up, put down the gun, and gimme some tropical flavored Bubblicious"
"...And some Skittles"
"You'll shoot your eye out, kid"
"Hakuna matata!"
"Here's... JHONNY!"
"THIS, IS, SPARTA!" (I'm so sorry)
"...I must break you"
"...Cocainum"
"No... I AM your father!"
"Who said that? WHO THE FUCK SAID THAT?!"
"Bullshit, show me your real war face!"
"WHAT IS YOUR MAJOR MALFUNCTION, NUMBNUT?!"

**Comment by nexusmrsep** [comment]
@Epictyphlosion some are duplicate, some are meh. I'd be blunt about it - I don't want to add those that break the immersion for me. But I'll add most of them.

 @Xhuis and @DracoGriffin - Crazy or not, its not a dumpster of ideas, and I want to hold some level I feel comfortable with, pure chaos is not fun, planned chaos is. About 10% of the quotes I ever considered did not met those conditions. If it fits the sceanario and sounds good then ok, but if it  makes me cringe becouse it sounds like from another story, then no, thanks, I'll pass.     There are more better alternatives.

**Comment by nexusmrsep** [comment]
Ok, unless there are more requested quotes, I consider this done and ready for merging.

**Comment by Brambor** [comment]
Lines 506 to 517 contain the same quote (are duplicates).

> Don't call me Shirley.

**Pull request merged**